plugins/systemd/systemd.plugin.zsh

```diff
@@ -8,4 +8,9 @@ sudo_commands=(
   link load cancel set-environment unset-environment)
 
 for c in $user_commands; do; alias sc-$c="systemctl $c"; done
-for c in $sudo_commands; do; alias sc-$c="sudo systemctl $c"; done
+# Do not use sudo for root user
+if [ "`id -u`" = "0" ]; then
```

Comment:
You should use `$UID` here instead - avoids a subshell: `if [[ $UID == 0 ]]; then`